lib/screens/send_request/bloc/send_request_state.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:equatable/equatable.dart';
 
 abstract class SendRequestState extends Equatable {
-  final String message;
+  String message;
```

Comment:
Removing final is not a proper solution.
I would suggest to use a resetEvent which yields initalState once message has been listened to.

Reply:
@Gaurav192 thanks for the suggestion, updated the pr can you give it a quick look? Thanks